keyloader/keyloader.go

```diff
@@ -261,52 +320,34 @@ func LoadKeyFromStore(identifier string, store *configstore.Store) (symmecrypt.K
 		return nil, fmt.Errorf("ambiguous config: several encryption keys conflicting for '%s'", identifier)
 	}
 
-	comp := symmecrypt.CompositeKey{}
-
-	hadNonSealed := false
+	var cfgs []*KeyConfig
 
 	for _, item := range items.Items {
-
 		i, err := item.Unmarshaled()
 		if err != nil {
 			return nil, err
 		}
-		var ref symmecrypt.Key
 		cfg := i.(*KeyConfig)
```

Comment:
I don't remember is this cast safe or should we `cfg, ok := (i.*KeyConfig)` ?

Reply:
It is safe (`Unmarshaled` returns either err or the type you supply)